packages/agent/src/certificate.test.ts

```diff
@@ -20,6 +19,24 @@ function pruned(str: string): ArrayBuffer {
   return fromHex(str);
 }
 
+function bufferEqualityTester(a: unknown, b: unknown): boolean | undefined {
```

Comment:
Again, is duplicative of other buffer utils that we already have

Reply:
huh... I had understood returning `undefined` was important to tell Jest to use a different equality tester, but this works:
```
(expect as any).addEqualityTesters([bufEquals]);
```

Thanks!